src/scenes/Event/components/EventTxHistory/resultHistory.js

```diff
@@ -66,19 +68,22 @@ export default class EventResultHistory extends Component {
               </TableRow>
             </TableHead>
             <TableBody>
-              {_.map(sortedOracles, (oracle, index) => (
-                <TableRow key={`result-${index}`} selected={index % 2 === 1}>
-                  <TableCell padding="dense">{getShortLocalDateTimeString(oracle.endTime)}</TableCell>
-                  <TableCell padding="dense">{this.getTypeText(oracle, index)}</TableCell>
-                  <TableCell padding="dense">
-                    {index !== sortedOracles.length - 1 && index !== 0
-                      ? `#${oracle.resultIdx + 1} ${oracle.options[oracle.resultIdx]}`
-                      : ''
-                    }
-                  </TableCell>
-                  <TableCell padding="dense">{`${_.sum(oracle.amounts)} ${oracle.token}`}</TableCell>
-                </TableRow>
-              ))}
+              {_.map(sortedOracles, (oracle, index) => {
+                const invalidOption = localizeInvalidOption(oracle.options[oracle.resultIdx], intl);
```

Comment:
can we move the `invalid` logic up here for readability?
```js
let invalidOption = oracle.options[oracle.resultIdx]
if (oracle.options[oracle.resultIdx] === 'Invalid') invalidOption = localizeInvalidOption(invalidOption, intl);

{(index !== sortedOracles.length - 1 && index !== 0) ? invalidOption : ''}
```

Reply:
i don't think its more readable. invalidOption should only means 'invalid', your code contains all the options.